email-report.html

```diff
@@ -0,0 +1,3672 @@
+18:45:02,601 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
```

Comment:
Was this file supposed to be checked in?  It looks as though some test output was captured to disk, then accidentally added to a git commit (one of the reasons why ```git commit -a``` is a bit of an anti-pattern...).

Reply:
nope :-) TY for spotting it